core/http/views/traces.html

```diff
@@ -0,0 +1,184 @@
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <template x-for="(trace, index) in traces" :key="index">
+                            <tr class="hover:bg-[var(--color-bg-secondary)]/50 border-b border-[var(--color-bg-secondary)] transition-colors">
+                                <td class="p-2" x-text="trace.request.method"></td>
+                                <td class="p-2" x-text="trace.request.path"></td>
+                                <td class="p-2" x-text="trace.response.status"></td>
+                                <td class="p-2 text-right">
+                                    <button @click="showDetails(index)" class="text-[var(--color-primary)]/60 hover:text-[var(--color-primary)] hover:bg-[var(--color-primary)]/10 rounded p-1 transition-colors">
+                                        <i class="fas fa-eye text-xs"></i>
+                                    </button>
+                                </td>
+                            </tr>
+                        </template>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+
+        <!-- Details Modal -->
+        <div x-show="selectedTrace !== null" class="fixed inset-0 bg-black/50 flex items-center justify-center z-50" @click="selectedTrace = null">
+            <div class="bg-[var(--color-bg-secondary)] rounded-lg p-6 max-w-4xl w-full max-h-[90vh] overflow-auto" @click.stop>
+                <div class="flex justify-between mb-4">
+                    <h2 class="h3">Trace Details</h2>
+                    <button @click="selectedTrace = null" class="text-[var(--color-text-secondary)] hover:text-[var(--color-text-primary)]">
+                        <i class="fas fa-times"></i>
+                    </button>
+                </div>
+                <div class="grid grid-cols-2 gap-4">
+                    <div>
+                        <h3 class="text-lg font-semibold mb-2">Request Body</h3>
+                        <div id="requestEditor" class="h-96 border border-[var(--color-primary-border)]/20"></div>
+                    </div>
+                    <div>
+                        <h3 class="text-lg font-semibold mb-2">Response Body</h3>
+                        <div id="responseEditor" class="h-96 border border-[var(--color-primary-border)]/20"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </div>
+
+    {{template "views/partials/footer" .}}
+
+</div>
+
+<!-- CodeMirror -->
+<link rel="stylesheet" href="static/assets/codemirror.min.css">
+<script src="static/assets/codemirror.min.js"></script>
+<script src="static/assets/javascript.min.js"></script>
+
+<!-- Styles from model-editor -->
+<style>
+.CodeMirror {
+    height: 100% !important;
+    font-family: monospace;
+}
+</style>
+
+<script>
+function tracesApp() {
+    return {
+        traces: [],
+        selectedTrace: null,
```

Comment:
should we also have an "export" to save as a jsonfile or either have a dump of the whole memory?